modules/registry-common-terms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * registry/index.adoc
+
+:_content-type: REFERENCE
+[id="openshift-registry-common-terms_{context}"]
+= Common registry terms
+
+This glossary defines common terms that are used in the registry content. 
+
+container::
+Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers anywhere, from a data center to a public or private cloud to your local host.
+
+Image Registry Operator::
+The Image Registry Operator runs in the `openshift-image-registry` namespace, and manages the registry instance in that location.
+
+image repository::
+An image repository is a collection of related container images and tags identifying images.
+
+mirror registry::
+A registry that holds the mirror of {product-title} images.
+
+namespace::
+A namespace isolates groups of resources within a single cluster.
+
+{product-title} registry::
+{product-title} registry is the registry provided by {product-title} to manage images.
+
+pod::
+The pod is the smallest logical unit in Kubernetes. A pod is comprised of one or more containers to run in a worker node.
+
+private registry::
+A registry is a server that implements the container image registry API. A private registry is a registry that requires authentication to allow users access to its contents.
+
+public registry::
+A registry is a server that implements the container image registry API. A public registry is a registry that serves its contently publicly.
+
+Quay.io::
+A public Red Hat Quay Container Registry instance provided and maintained by Red Hat, that serves most of the container images and Operators to {product-title} clusters.
+
+registry authentication::
+To push and pull images to and from private image repositories, The registry needs to authenticate its users with credentials.
+
+route::
+Exposes a service to allow for network access to pods from users and applications outside the {product-title} instance.
+
+scale down::
+Decreasing the number of replicas.
```

Comment:
`To decrease the number of replicas`
Any particular reason why continuous tense is used?

Reply:
acked